src/Particle/ParticleAttrib.hpp

```diff
@@ -186,14 +186,15 @@ namespace ippl {
             "ParticleAttrib::gather", policy_type(0, *(this->localNum_mp)),
             KOKKOS_CLASS_LAMBDA(const size_t idx) {
                 // find nearest grid point
-                vector_type l                        = (pp(idx) - origin) * invdx + 0.5;
+                vector_type l                        = (pp(idx) - origin) * invdx + 1.0;
```

Comment:
Why is this changed?

Reply:
This is due to the ParticleAttrib::scatter and gather interplating to the index cube (i,j,k) -- (i-1,j-1,k-1) which is very confusing, and I add an offset of 1 here to change it to (i,j,k) to (i+1,j+1,k+1). Also I don't do centering (it's an totally meaningless offset to the particle positions anyway)

Reply:
Confusing in what way? This is sort of a standard way in the community to do linear/cloud-in-cell interpolation from particles to cell-centered meshes. You can have a look at other community code https://github.com/AMReX-Codes/amrex/blob/development/Src/Particle/AMReX_ParticleInterpolators.H#L239 . I am not saying we should only follow them but you should have a good reason if you want to change the norm. Also since the scatter and gather are not only limited to this project if we want to adopt this all the previous Alpine results (ideally all other tests/results which have used the previous scatter and gather) have to be done again and the results have to be verified. I am not sure if we want to do that.

Reply:
I guess that's true, I should rather fit the rest of my scatter functions to work with ippls gather.

Reply:
Yeah that would be great.. Because there are also fillHalo and accumulateHalo functions for periodic BCs written assuming the previous scatter/gather and I am not sure if they work with this too.